Automattic/wp-calypso
# Domains: Migrate Office365 to DNS Templates

We're introducing DNS Templates which makes it easy to apply a set of DNS Records by passing only the needed variables.

Test:
1. Domain Management
2. Pick a registered domain
3. Go to DNS Editor
4. At the bottom click Office 365
5. Make sure the records get applied
